### DISCUSSION TITLE
Need little assist for setting in Vlad's webui

### DISCUSSION THREAD

**Discussion by josh5671** [vladmandic/sdnext#417] [Q&A] [answered]
Kinda use automatic1111's webui from day 1 it appear and switching to Vlad's webui today.
(have 0 knowledge on python, might be not sure what i am talking about, please correct me if i am wrong)

I am running 3070 with Torch2+CU118 on 1111, with --no-half-vae --opt-sdp-no-mem-attention
I able to generate 3 batch size 30 step 480x640 and 20 step highres 960x1280 (i might be doing it wrong, but so far i always doing this)

On Vlad i unable to generate with same setting.
It just straight out of vram.

> 
> ╭───────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────╮
> │ E:\automatic\modules\call_queue.py:58 in f                                                                           │
> │                                                                                                                      │
> │    57 │   │   │   │   pr.enable()                                                                                    │
> │ ❱  58 │   │   │   res = list(func(*args, **kwargs))                                                                  │
> │    59 │   │   │   if shared.cmd_opts.profile:                                                                        │
> │                                                                                                                      │
> │ E:\automatic\modules\call_queue.py:38 in f                                                                           │
> │                                                                                                                      │
> │    37 │   │   │   try:                                                                                               │
> │ ❱  38 │   │   │   │   res = func(*args, **kwargs)                                                                    │
> │    39 │   │   │   finally:                                                                                           │
> │                                                                                                                      │
> │                                               ... 13 frames hidden ...                                               │
> │                                                                                                                      │
> │ E:\automatic\repositories\stable-diffusion-stability-ai\ldm\modules\diffusionmodules\model.py:132 in forward         │
> │                                                                                                                      │
> │   131 │   │   h = self.norm1(h)                                                                                      │
> │ ❱ 132 │   │   h = nonlinearity(h)                                                                                    │
> │   133 │   │   h = self.conv1(h)                                                                                      │
> │                                                                                                                      │
> │ E:\automatic\venv\lib\site-packages\torch\nn\functional.py:2059 in silu                                              │
> │                                                                                                                      │
> │   2058 │   │   return torch._C._nn.silu_(input)                                                                      │
> │ ❱ 2059 │   return torch._C._nn.silu(input)                                                                           │
> │   2060                                                                                                               │
> ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
> OutOfMemoryError: CUDA out of memory. Tried to allocate 1.76 GiB (GPU 0; 8.00 GiB total capacity; 5.95 GiB already
> allocated; 0 bytes free; 6.52 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting
> max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

And time to time i have seen it try to allocate 10gb vram while my gpu only have 8gb.

*
PS: any extra detail i gonna to provide so you guys able to provide asist?

**Comment by vladmandic** [comment]
`--no-half-vae` and `--opt-sdp-no-mem-attention` don't exist here, you need to set values via UI settings.

**Reply by josh5671** [reply]
thanks for almost instantly reply.

yes, the no-half-vae and opt-sdp-no-mem-attention has been set in Cuda Setting
the cuda memory run out still there. 
![123456](https://user-images.githubusercontent.com/9864414/233848355-747f7b86-77f5-48f7-970b-09f47bbf5e85.jpg)
![123456](https://user-images.githubusercontent.com/9864414/233848357-fafaab49-bde6-4bba-b369-9263746e4068.png)

**Comment by vladmandic** [comment, marked as answer]
this may be just borderline - you were running very near the limit and torch 2.0 is just a little bit more memory hungry. the allocation values do not seem horrible on their own.

**Reply by josh5671** [reply]
I see. kinda agree i touch the hard borderline of your of your webui.
For now will switch back to 1111's webui (just copy the model,extension and output back), will come back a week or weeks later, since Vlad you literally update it daily or between days.
Maybe my next try, all will be working fine.
